manifests/params.pp

```diff
@@ -33,6 +33,35 @@
         ],
         'maxconn' => '8000'
       }
+        $config_file      = '/etc/haproxy/haproxy.cfg'
+      }
```

Comment:
Indentation is weird here, I think you wanted the above to lines to be 2 space to the left?